build.sh

```diff
@@ -81,6 +81,14 @@ for BINARY in $BINARIES; do
     error "Unable to find output"
     exit 1
   fi
+  
+  # MINIFY set to false by default
+  MINIFY=${MINIFY:-"false"}
+  if [ "$MINIFY" = "true" ]; then
+    info "Minifying ${OUTPUT}..."
+    strip "${OUTPUT}"
```

Comment:
stdout needs to be redirected to stderr as stdout is being used to identify what files to add to zip in entrypoint
```suggestion
    strip "${OUTPUT}" >&2
    upx "${OUTPUT}" >&2
```